### DISCUSSION TITLE
Flaky on the component will automatically re-render through the bound blur event handler

### DISCUSSION THREAD

**Discussion by Daily-Prophet** [cypress-io/cypress#15784] [Component Testing]
## Context
I used a [Fluent UI menu component](
https://fluentsite.z22.web.core.windows.net/0.52.1/components/menu-button/definition) in my project and I want to test click its submenu item. 
The submenu seems like bind with some kind of blur event listener. Once move mouse cursor outside the browser, the submenu will disappear.  
## Issue  
Because I keep Cypress focused when working on a test, everything is working fine when I test it locally, 
**But when CI test, it will flaky - Cypress can't open the submenu anymore and can't get its items, since the test brower may lost its focus status.**
Do you have any suggestions to resolve this flaky? 
## GIF description
![flaky](https://user-images.githubusercontent.com/25863604/113504985-87aaf000-956e-11eb-94b7-923ab9454137.gif)

**Comment by lmiller1990** [comment]
Hi! Sorry for the slow reply. I'll be focusing on answering questions here from now on. 

Are you using the component testing runner (`yarn cypress open-ct` or `run-ct`)?

Hard to say without seeing some code - ideally, if you can provide a repo, I can take a look. If not, can you at least share the component code and test code, so I can reproduce it?
